__tests__/errors.test.js

```diff
@@ -105,3 +105,66 @@ describe("POST /api/articles/:article_id/comments", () => {
         })
     })
 })
+
+describe("PATCH /api/articles/:article_id", () => {
+    test("400 responds when valid path but invalid id", () => {
+        return request(app)
+        .get("/api/articles/invalidId")
+        .expect(400)
+        .then(( { body }) => {
+            expect(body.msg).toBe("invalid input");
+        })
```

Comment:
Be careful, we still need to send a body here. We don't want 2 errors being tested at once as it could give us false positives :)

Reply:
This should be handled by SQL error